tests/framework/utils.py

```diff
@@ -30,3 +34,94 @@ def search_output_from_cmd(cmd: str,
 
     raise RuntimeError("Could not find '%s' in output for '%s'" %
                        (find_regex.pattern, cmd))
+
+
+def get_files_from(find_path: str,
```

Comment:
nit: either rename the function or add a flag to suggest that it traverses `find_path` recursively